packages/app/pages/course/[cid]/today.tsx

```diff
@@ -68,8 +68,9 @@ export default function Today(): ReactElement {
     mutateCourse();
   };
 
+  console.log(course, profile);
   const queueCheckedIn = course?.queues.find((queue) =>
-    queue.staffList.find((staff) => staff.id === profile.id)
+    queue.staffList.find((staff) => staff.id === profile?.id)
```

Comment:
race condition exacerbated by the zero downtime deployment, since requests are more likely to arrive out of order